pages/common/gist.md

```diff
@@ -0,0 +1,23 @@
+# gist
+
+> Upload code to https://gist.github.com.
+
+- Upload any number of files:
+
+`gist {{file.txt}} {{file2.txt}}`
+
+- Upload a private file with a description:
+
+`gist -p {{file.txt}} -d {{"A meaningful description"}}`
+
+- Read contents from STDIN with a name:
+
+`{{echo hello world}} | gist -f {{name.rb}}`
+
+- List public and private gists for authed user:
```

Comment:
I've changed the wording to make it clearer.

Also, by motivation in using "private" rather than "secret" is the `man gist` pages use the "private" term.

http://manpages.ubuntu.com/manpages/xenial/man1/gist-paste.1.html

Becuase the option is also "-p" for private, I supposed it would make it easier for the reader to relate them.